test/visibility/import/public/enablesUnqualified/importAndImport.chpl

```diff
@@ -0,0 +1,25 @@
+// Tests behavior when a module is imported from two different paths
+module A {
+  var x: int = 4;
+  proc foo() {
+    writeln("In A.foo()");
+  }
+}
+module B {
+  // Note: this import is private and thus does not impact the symbols available
```

Comment:
You'll want to update these comments :)

Reply:
Maintaining comments is more difficult than maintaining code...

Reply:
Ain't that the truth XD